src/components/ex-cta-box/cta-box.scss

```diff
@@ -0,0 +1,158 @@
+.cta {
+  @apply mt-component-spacing-small h-full;
+
+  &__text {
+    a {
+      @apply underline transition-all duration-300 ease-in-out;
+
+      &:hover {
+        @apply text-primary;
+      }
+    }
+  }
+
+  .cta__button {
+    @apply w-auto px-6 py-2 mx-auto mt-6 no-underline break-all;
+  }
+
+  &__title {
+    hyphens: auto;
+  }
+
+  &--background-color {
+    @apply bg-primary flex flex-col h-full px-8 py-6 mx-auto;
+
+    .cta__title {
+      @apply text-xl font-light text-center text-white uppercase;
+    }
+
+    .cta__text {
+      @apply flex-grow mt-6 text-center text-white;
+
+      a:hover {
+        @apply text-gray-3;
+      }
+    }
+
+    .cta__button {
+      @apply text-primary relative flex items-center text-left uppercase transition-all duration-100 ease-in-out bg-white;
+
+      &:hover {
+        @screen sm {
+          @apply bg-interactive-state-hover-button text-white no-underline;
+        }
+      }
+    }
+
+    &[data-bg-color='#AB0F1F'] {
+      background-color: rgba(171, 15, 31, 1);
+
+      .cta__button {
+        @apply text-black;
+
+        &:hover {
+          @screen sm {
+            @apply text-white;
+          }
+        }
+      }
+    }
+
+    &[data-bg-color='#B7BAB6'] {
+      background-color: rgba(183, 186, 182, 1);
+
+      .cta__title,
+      .cta__text {
+        @apply text-black;
+
+        a {
+          &:hover {
+            @apply text-interactive-state-hover-link;
+          }
+        }
+      }
+
+      .cta__button {
+        @apply text-black;
+
+        &:hover {
+          @screen sm {
+            @apply text-white;
+          }
+        }
+      }
+    }
+  }
+
+  .cta--background-image {
+    @apply relative flex flex-col h-full px-2 py-6 mx-auto text-center uppercase;
+
+    .cta__container {
+      @apply px-4 mx-auto;
+    }
+
+    .cta__title {
+      @apply font-headline text-primary text-xl font-light;
+    }
+
+    .cta__text {
+      @apply font-menu text-default flex-grow font-light text-black;
+    }
+
+    .cta__button {
+      @apply bg-primary inline-block text-white transition-all duration-100 ease-in-out;
+
+      &:hover {
+        @screen sm {
```

Comment:
There is no mouse-ever effect on mobile. It should work on mobile as well.

Reply:
This comment is in the wrong place. I refer to the white buttons at the end of the box.

Reply:
Activated hover for mobile

Reply:
Good start. The CTA in red color as black font color on navy background. Should be white background:

![image](https://user-images.githubusercontent.com/88790097/134925782-0281292f-ed35-429f-aed5-d4d7720d6fd8.png)

Reply:
should be fixed